core/pkg/opencost/summaryallocation_json.go

```diff
@@ -35,6 +35,16 @@ func (sa *SummaryAllocation) ToResponse() *SummaryAllocationResponse {
 		return nil
 	}
 
+	// if the efficiency has already been set,
+	// prefer that since it has been calculated in the DB
+	// and matches the sorting criteria more closely
```

Comment:
Minor nit. Would it be best not to refer to a DB here? Generalize this to any consumer of the SummaryAllocation struct?